Executing reingest command fails with a segmentation violation

### What version are you using?

horizon 1.6.0 (installed from Ubuntu repo)

### What did you do?

execute
`$ stellar-horizon db reingest range 30759871 30842880 --db-url="postgres://horizon:XXX@XXX/horizon" --network-passphrase="Public Global Stellar Network ; September 2015" --stellar-core-db-url="postgres://horizon:XXX@XXX/stellarcore" --stellar-core-url="http://XXX:11626"`

### What did you expect to see?

reingest running

### What did you see instead?

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xd064e8]

goroutine 1 [running]:
github.com/stellar/go/services/horizon/cmd.glob..func5(0x18e0e40, 0xc0002e65a0, 0x2, 0x6)
        /var/lib/jenkins/jobs/stellar-horizon-public/workspace/services/horizon/cmd/db.go:239 +0x3b8
github.com/spf13/cobra.(*Command).execute(0x18e0e40, 0xc0002e64e0, 0x6, 0x6, 0x18e0e40, 0xc0002e64e0)
        /var/lib/jenkins/jobs/stellar-horizon-public/workspace/vendor/github.com/spf13/cobra/command.go:603 +0x23b
github.com/spf13/cobra.(*Command).ExecuteC(0x18e14a0, 0x43ed7a, 0x17f8d60, 0xc000000300)
        /var/lib/jenkins/jobs/stellar-horizon-public/workspace/vendor/github.com/spf13/cobra/command.go:689 +0x2bc
github.com/spf13/cobra.(*Command).Execute(...)
        /var/lib/jenkins/jobs/stellar-horizon-public/workspace/vendor/github.com/spf13/cobra/command.go:648
github.com/stellar/go/services/horizon/cmd.Execute()
        /var/lib/jenkins/jobs/stellar-horizon-public/workspace/services/horizon/cmd/root.go:442 +0x31
main.main()
        /var/lib/jenkins/jobs/stellar-horizon-public/workspace/services/horizon/main.go:6 +0x20
```

### Investigation
I tried with horizon 1.5.0 (installed from Ubuntu repo) and got the error that the archive url's where not filled in.
When running the 1.6.0 version again with the archive url's filled in, everything works fine

### Guess
I think the archive url argument check has been removed for the reingest command

--- a/services/horizon/cmd/db.go
+++ b/services/horizon/cmd/db.go
@@ -232,7 +232,7 @@ var dbReingestRangeCmd = &cobra.Command{
 
 		if parallelWorkers < 2 {
 			system, systemErr := expingest.NewSystem(ingestConfig)
-			if err != nil {
+			if systemErr != nil {
 				log.Fatal(systemErr)
 			}
 
@@ -243,7 +243,7 @@ var dbReingestRangeCmd = &cobra.Command{
 			)
 		} else {
 			system, systemErr := expingest.NewParallelSystems(ingestConfig, parallelWorkers)
-			if err != nil {
+			if systemErr != nil {
 				log.Fatal(systemErr)
 			}
 

